test/create-run-publish-app.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Yes, my I have looked at his PR and intent to do the same here. But I already had enough of this done that it seemed worthwhile to do a PR and then change it instead of waiting. More coverage sooner seemed like a good idea.